### PR TITLE
[FEAT] Beekeeper Hat Boost

### PR DESCRIPTION
**Pull request by Rofrtd**
# Description

Beekeeper hat gives 20% speed boost to honey production

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by c-py — APPROVED** [review]
LGTM!

**Comment by brian-man** [comment]
@Rofrtd 
Needs to be added to `BUMPKIN_ITEM_BUFF_LABELS` also.  Right?

**Review by eliasSFL — CHANGES_REQUESTED** [review]
Would be good to change to this

**Review by eliasSFL — CHANGES_REQUESTED** [review]
Would be good to standardise the Beekeeper Hat boost labels to be the same as other time reduction labels.

Edit: Idk how to show suggested code changes in github lol

File: src/features/game/types/bumpkinItemBuffs.ts [hunk 1]
```diff
@@ -194,4 +194,10 @@ export const BUMPKIN_ITEM_BUFF_LABELS: Partial<Record<BumpkinItem, BuffLabel>> =
       boostTypeIcon: powerup,
       boostedItemIcon: ITEM_DETAILS["Red Pansy"].image,
     },
+    "Beekeeper Hat": {
+      shortDescription: translate("description.beekeeper.hat.boost"),
+      labelType: "vibrant",
```
Comment (eliasSFL) [inline src/features/game/types/bumpkinItemBuffs.ts:199]:
```suggestion
      labelType: "info",
```

File: src/features/game/types/bumpkinItemBuffs.ts [hunk 2]
```diff
@@ -194,4 +194,10 @@ export const BUMPKIN_ITEM_BUFF_LABELS: Partial<Record<BumpkinItem, BuffLabel>> =
...
+      boostTypeIcon: lightning,
```
Comment (eliasSFL) [inline src/features/game/types/bumpkinItemBuffs.ts:200]:
```suggestion
      boostTypeIcon: SUNNYSIDE.icons.stopwatch,
```

**Review by eliasSFL — CHANGES_REQUESTED** [review]
@Rofrtd Suggest standardising the boost label to match other time based boosts as well

File: src/features/game/types/bumpkinItemBuffs.ts (hunk 2 above)
Comment (eliasSFL) [inline src/features/game/types/bumpkinItemBuffs.ts:200]:
```suggestion
      labelType: "info",
      boostTypeIcon: SUNNYSIDE.icons.stopwatch,
```

**Pull request merged**